be/src/exec/pipeline/hashjoin/hash_join_build_operator.cpp

```diff
@@ -103,9 +103,13 @@ Status HashJoinBuildOperator::set_finishing(RuntimeState* state) {
 
     // add partial filters generated by this HashJoinBuildOperator to PartialRuntimeFilterMerger to merge into a
     // total one.
-    auto status = _partial_rf_merger->add_partial_filters(merger_index, ht_row_count, std::move(partial_in_filters),
-                                                          std::move(partial_bloom_filter_build_params),
-                                                          std::move(partial_bloom_filters));
+    StatusOr<bool> status;
+    {
+        SCOPED_TIMER(_join_builder->build_metrics().build_runtime_filter_timer);
+        status = _partial_rf_merger->add_partial_filters(merger_index, ht_row_count, std::move(partial_in_filters),
```

Comment:
spillable hash join build need the same change

Reply:
> spillable hash join build need the same change

if no spilling happens, it will invoke HashJoinBuildOperator::set_finishing, seems no need to change?

Reply:
in spillable_hash_join_build_operator.cpp:
```
 auto merged = _partial_rf_merger->set_always_true();

    if (merged) {
```
merged was always true. it will cause set_collector exec mutli-times

Reply:
> in spillable_hash_join_build_operator.cpp:
> 
> ```
>  auto merged = _partial_rf_merger->set_always_true();
> 
>     if (merged) {
> ```
> 
> merged was always true. it will cause set_collector exec mutli-times

got u. seems we should change return type of set_always_true to StatusOr<bool>, I prefer to fix it in later PR.